File: Python/Personal/light.py
class light:
    def __init__(self,type,watts,lumens, working):
        self.type = type
        self.watts = watts
        self.lumens = lumens
        self.working = working
    
    def on(self):
        if self.working == 1:
            print('Light is on!')
            self.status=1
        else:
            print('The light is blown!')
            pass

    def off(self):
        print('Light is off!')
        self.status=0

oldLight = light('halogen',65,100,0)
newLight = light("led",5,250,1)

print(f'{newLight.lumens} Lumens')
newLight.on()
print(newLight.status)
print()
print(f'{oldLight.lumens} Lumens')
oldLight.on()
print(oldLight.working)

File: Python/ChatGPT (Only for fun!)/light.cs
using System;

public class Light
{
    public string Type;
    public int Watts;
    public int Lumens;
    public int Status;

    public Light(string type, int watts, int lumens)
    {
        Type = type;
        Watts = watts;
        Lumens = lumens;
        Status = 0;
    }

    public void On()
    {
        Console.WriteLine("Light is on!");
        Status = 1;
    }

    public void Off()
    {
        Console.WriteLine("Light is off!");
        Status = 0;
    }
}

public class Program
{
    public static void Main()
    {
        Light newLight = new Light("led", 5, 250);

        Console.WriteLine(newLight.Lumens);
        newLight.On();
        Console.WriteLine(newLight.Status);
    }
}
